l10n/en/firefox/whatsnew/whatsnew-sync.ftl

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/whatsnew/
+
+-brand-name-firefox = Firefox
+-brand-name-firefox-browser = Firefox Browser
```

Comment:
We should put these into a shared file, I'd suggest `l10n/en/brands.ftl`, and add that to the list of ftl files, appending at the end.

Reply:
And add a file-level comment there pointing to https://mozilla-l10n.github.io/styleguides/mozilla_general/index.html#brands-copyright-and-trademark, please? With a "Please read the trademarks localization guide" or so